web/src/app/administration/components/classification-types-selector/classification-types-selector.component.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { ClassificationTypesSelectorComponent } from './classification-types-selector.component';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import { NgxsModule, Store } from '@ngxs/store';
+import { ClassificationState } from '../../../shared/store/classification-store/classification.state';
+import { MatRadioButton, MatRadioGroup } from '@angular/material/radio';
+import { ClassificationsService } from '../../../shared/services/classifications/classifications.service';
+import { ClassificationCategoriesService } from '../../../shared/services/classification-categories/classification-categories.service';
+import { DomainService } from '../../../shared/services/domain/domain.service';
+import { MatRippleModule } from '@angular/material/core';
+
+export const classificationInitState = {
+  selectedClassificationType: 'Document',
+  classificationTypes: {
+    TASK: [],
+    DOCUMENT: []
+  }
+};
```

Comment:
For future reference, reusable or big json data can be added in store/mock-data.ts. This is added in other PR.